app/lib/features/chat/widgets/custom_input.dart

```diff
@@ -371,38 +371,22 @@ class __ChatInputState extends ConsumerState<_ChatInput> {
                 mainAxisSize: MainAxisSize.min,
                 mainAxisAlignment: MainAxisAlignment.spaceEvenly,
                 children: <Widget>[
-                  Padding(
-                    padding: const EdgeInsets.only(right: 10),
-                    child: InkWell(
-                      onTap: () => selectAttachment(
-                        context: context,
-                        onSelected: handleFileUpload,
-                      ),
-                      child: const Icon(
-                        Atlas.paperclip_attachment_thin,
-                        size: 20,
-                      ),
-                    ),
-                  ),
                   Flexible(
-                    child: Padding(
-                      padding: const EdgeInsets.symmetric(horizontal: 10),
-                      child: _TextInputWidget(
-                        roomId: widget.roomId,
-                        controller: textController,
-                        chatFocus: chatFocus,
-                        onSendButtonPressed: () => onSendButtonPressed(ref),
-                        isEncrypted: isEncrypted,
-                        onTyping: widget.onTyping,
-                      ),
+                    child: _TextInputWidget(
+                      roomId: widget.roomId,
+                      controller: textController,
+                      chatFocus: chatFocus,
+                      onSendButtonPressed: () => onSendButtonPressed(ref),
+                      isEncrypted: isEncrypted,
+                      onTyping: widget.onTyping,
                     ),
                   ),
                   ValueListenableBuilder<bool>(
                     valueListenable: _isInputEmptyNotifier,
                     builder: (context, isEmpty, child) {
                       return !isEmpty
                           ? renderSendButton(context, roomId)
-                          : const SizedBox();
+                          : renderAttachmentPinButton();
```

Comment:
So we're essentially hiding attachment button when user is typing text? This seems wrong. Attachment icon should always be visible...